x/concentrated-liquidity/swaps_test.go

```diff
@@ -265,11 +288,15 @@ func (s *KeeperTestSuite) TestCalcAndSwapOutAmtGivenIn() {
 			priceLimit:    sdk.NewDec(6056),
 			// expectedTokenIn:  5238677582.189386755771808942932776 + 4761322417.8106132444 = 10000000000.0000 = 10000.00 usdc
 			// expectedTokenOut: 998976.6183474263883566299269692777 + 865185.2591363751404579873403641 = 1864161.877 round down = 1.864161 eth
-			expectedTokenIn:  sdk.NewCoin("usdc", sdk.NewInt(10000000000)),
-			expectedTokenOut: sdk.NewCoin("eth", sdk.NewInt(1864161)),
-			expectedTick:     sdk.NewInt(320560),
-			newLowerPrice:    sdk.NewDec(5001),
-			newUpperPrice:    sdk.NewDec(6250),
+			expectedTokenIn:                  sdk.NewCoin("usdc", sdk.NewInt(10000000000)),
+			expectedTokenOut:                 sdk.NewCoin("eth", sdk.NewInt(1864161)),
+			expectedTick:                     sdk.NewInt(320560),
+			expectedLowerTickFeeGrowth:       DefaultFeeAccumCoins,
```

Comment:
It would be great to have some short comment explanations for each test case and each position about why these are the values we are expecting.

I'm working on swap fees right now and finding such comments extremely valuable

Reply:
Added!